apps/tng-dictionary/src/utilities/matchers/stringIncludes.ts

```diff
@@ -1,3 +1,3 @@
 export default (input: string | null, textToMatch: string) =>
-  // No match if the input is falsey
```

Comment:
@aaron-plahn

Reply:
Let's rename this file and its import (in VocabularyListIndex and TermIndex) `stringIncludesCaseInsenstivie`.